scripts/ci_build_and_test_in_container.sh

```diff
@@ -174,7 +185,8 @@ or_die python3 scripts/config-build.py \
                -ip ${GEOSX_DIR} \
                --ninja \
                -DBLT_MPI_COMMAND_APPEND='"--allow-run-as-root;--oversubscribe"' \
-               -DGEOSX_INSTALL_SCHEMA=${GEOSX_INSTALL_SCHEMA} \
+               -DGEOSX_INSTALL_SCHEMA=$([[ "$*" == *--disable-schema-deployment* ]] && echo 0 || echo 1) \
```

Comment:
We normally have the `--no-install-schema` option that populates the `GEOSX_INSTALL_SCHEMA` variable.
Why lead you into this change?

Reply:
I didn't. This was here from @tbeltzun